jujugui/static/gui/src/app/components/deployment-flow/cloud/cloud.js

```diff
@@ -52,6 +52,9 @@ class DeploymentCloud extends React.Component {
         if (cloudList.length === 1) {
           this.props.setCloud(cloudList[0]);
         }
+        if (this.props.setCloudCount) {
```

Comment:
Why not making setCloudCount required?

Reply:
Ah, because we share this component in the profile as well and there we handle the change cloud button a bit differently (some other props are also not required for the profile).